src/templates/admin/cms/nav.html

```diff
@@ -15,8 +18,18 @@
 {% block body %}
     <div class="large-6 columns">
         <div class="box">
+            {% include "admin/elements/translations/form_tabs.html" with object=nav_item_to_edit %}
             <form method="POST">
                 {% csrf_token %}
+                <small>
+                    <p>To create a navigation link to a CMS page:</p>
+                    <ul>
+                        <li>{% if settings.URL_CONFIG == 'path' and request.journal %}Prefix the link with {{ request.journal.code }}/{% endif %}site/</li>
```

Comment:
I'm thinking we should do this when rendering the navigation rather than having users type in the journal code